src/util/axis.js

```diff
@@ -12,11 +12,13 @@ import { Vector4 } from "three";
 export const setOrigin = function (vec, dimensions, origin) {
   if (+dimensions === dimensions) {
     dimensions = [dimensions];
+  } else {
+    dimensions = Array.from(dimensions);
   }
-  const x = Array.from(dimensions).includes(1) ? 0 : origin.x;
```

Comment:
while debugging a different problem that turned out to be user error, I introduced this mild cleanup.